docs/logs/customizing-the-sdk/Program.cs

```diff
@@ -39,15 +42,39 @@ public static void Main()
 
         var logger = loggerFactory.CreateLogger<Program>();
 
+        // unstructured log
         logger.LogInformation("Hello Information");
         logger.LogWarning("Hello Warning");
         logger.LogError("Hello Error");
 
+        // structured log with template
+        logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
+
+        // structured log with strong type
```

Comment:
How is this related to the feature being documented?

Reply:
I included it here for completeness, because the type is part of the state object. 
Although it's not handled well by the ConsoleExporter.

I can omit this and focus on the simpler example using the message template